be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -573,7 +573,7 @@ Status ColumnReader::_load_inverted_index_index(
                         "create StringTypeInvertedIndexReader error: {}", e.what());
             }
         }
-    } else if (is_numeric_type(type)) {
+    } else if (is_numeric_type(type) || is_ip_type(type)) {
```

Comment:
is_numeric_type() should be the same as field_is_numeric_type() and is_ip_type() can be removed here.

Reply:
ok I will make the same